packages/dds/test-dds-utils/src/index.ts

```diff
@@ -14,9 +14,11 @@ export type {
 	DDSFuzzSuiteOptions,
 	DDSFuzzTestState,
 	DDSFuzzHarnessEvents,
+	HandleCreated,
 	Synchronize,
 } from "./ddsFuzzHarness.js";
 export { createDDSFuzzSuite, defaultDDSFuzzSuiteOptions, replayTest } from "./ddsFuzzHarness.js";
+export { DDSFuzzHandle } from "./ddsFuzzHandle.js";
```

Comment:
i don't think this needs to be exported outside this package, so doesn't need to be in the index